From 501b71d989f384fd5342c3783308a7ae469414a1 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Fri, 10 Jan 2025 17:11:24 +0100
Subject: [PATCH 1446/1695] dts: stm32: remove always-on from vdd33ucpd in
 stm32mp235f-dk.dtsi

Remove always-on property from "vdd33ucpd" node and its parent "vdd3v3_usb"
On stm32mp235f-dk board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git bbad2fe98d6cf7f5c504e3dadd0ea040c3970528 dts: stm32: remove always-on from vdd33ucpd in stm32mp235f-dk.dtsi]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I1aaedbc175f73bfea7973346ee9be3cec555662d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/428240
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp235f-dk.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp235f-dk.dts b/core/arch/arm/dts/stm32mp235f-dk.dts
index 1c5aa27ae..a1f8869db 100644
--- a/core/arch/arm/dts/stm32mp235f-dk.dts
+++ b/core/arch/arm/dts/stm32mp235f-dk.dts
@@ -273,7 +273,6 @@
 				regulator-name = "vdd3v3_usb";
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
-				regulator-always-on;
 				regulator-over-current-protection;
 			};
 			v5v_hdmi: ldo5 {
@@ -337,7 +336,6 @@
 		regulator-min-microvolt = <3300000>;
 		regulator-max-microvolt = <3300000>;
 		vdd33ucpd-supply = <&vdd3v3_usb>;
-		regulator-always-on;
 	};
 
 	vdda18adc: vdda18adc {
-- 
2.47.2

